kaiax/gasless/impl/init.go

```diff
@@ -55,6 +57,7 @@ func (g *GaslessModule) Init(opts *InitOpts) error {
 	g.allowedTokens = map[common.Address]bool{
 		common.HexToAddress("0xabcd"): true,
```

Comment:
Excuse me, this may be unrelated to this PR, but how do I set the actual swapRouters and allowedTokens values?

Reply:
You can set them by only hard-code now.

Reply:
@shiki-tak We should deploy the contracts before release. I think we should use deterministic deploy so the address is the same across chains.

Reply:
It's also good to set addresses in chain config.

Reply:
Actually, even if we used deterministic deploy, WKAIA address is different between mainnet and testnet, so Router will have different addresses.
ChainConfig sounds good.